ftd/src/interpreter/main.rs

```diff
@@ -1100,19 +1100,33 @@ impl Document {
             .collect_vec()
     }
 
-    pub fn get_redirect(&self) -> Option<(String, i32)> {
+    pub fn get_redirect(&self) -> Result<Option<(String, i32)>, super::Error> {
```

Comment:
Use this Result type
```suggestion
    pub fn get_redirect(&self) -> fastn_core::Result<Option<(String, i32)> {
```

Reply:
changed Result -> ftd::interpreter::Result